WindowBoard.cpp

```diff
@@ -17,34 +17,49 @@
 #include <Rect.h>
 #include <String.h>
 #include <StringView.h>
+#include <TranslationUtils.h>
 
 GameWindow::GameWindow(WindowBoard *master)
 	:
 	BWindow(BRect(100, 100, 500, 400), "Haiku2048", B_TITLED_WINDOW, 0),
 	fMaster(master)
-{
-	BButton *newGameButton = new BButton("newgame", "New Game",
-		new BMessage(H2048_NEW_GAME));
+{	
+	fIconNew  = BTranslationUtils::GetBitmap('PNG ', "icon_new.png");
```

Comment:
Using vector icons instead of PNG would be preferred.
See http://xref.plausible.coop/source/search?q=GetVectorIcon&defs=&refs=&path=src%2Fapps&hist=&type=&project=haiku for examples in existing Haiku apps.

Reply:
Also, the icon should be loaded from the resources, so we have just the app's one binary to distribute with no additional files to put somewhere. Search for "LoadResource".

Reply:
Here it is: 
![virtualbox_haiku_31_12_2017_22_37_11](https://user-images.githubusercontent.com/19305859/34462781-bc921168-ee7d-11e7-91b4-857ffc039876.png)